.github/workflows/docs.yml

```
@@ -15,6 +15,8 @@ jobs:
       - uses: actions/checkout@v3
 
       - uses: actions/setup-python@v4
+        with:
+          python-version: "3.x"
```

I think this version of the action supports SemVer qualifiers, so maybe this?

```suggestion
          python-version: ">= 3.7"
```